django/cantusdb_project/main_app/templates/source_detail.html

```diff
@@ -210,33 +210,31 @@ <h4>{{ source.siglum }}</h4>
                             Notation: <b><a href="#">{{ source.notation.all.first.name }}</a></b>
                             <br>
                         {% endif %}
-                        {% if source.inventoried_by.all %}
+                        {% if source.inventoried_by_u.all %}
                             Inventoried by:
                             <ul>
-                                {% for editor in source.inventoried_by.all %}
+                                {% for editor in source.inventoried_by_u.all %}
                                     <li>
-                                        <a href={{ editor.get_absolute_url }}><b>{{ editor.given_name }}
-                                            {{ editor.family_name }}</b></a><br>
+                                        <a href={{ editor.get_absolute_url }}><b>{{ editor.full_name }}</b></a><br>
                                         {{ editor.institution|default_if_none:"" }}
                                     </li>
                                 {% endfor %}
                             </ul>
                         {% endif %}
-                        {% if source.proofreaders.all %}
-                            Proofreader{{ source.proofreaders.all|pluralize }}:
+                        {% if source.proofreaders_u.all %}
+                            Proofreader{{ source.proofreaders_u.all|pluralize }}:
                             <br>
                             <ul>
-                                {% for editor in source.proofreaders.all %}
+                                {% for editor in source.proofreaders_u.all %}
                                     <li>
-                                        <a href={{ editor.get_absolute_url }}><b>{{ editor.given_name }}
-                                            {{ editor.family_name }}</b></a><br>
+                                        <a href={{ editor.get_absolute_url }}><b>{{ editor.full_name }}</b></a><br>
                                     </li>
                                 {% endfor %}
                             </ul>
                         {% endif %}
                         {{ source.indexing_notes|default_if_none:"" }}
                         <br>
-                        Contributor: <a href="">CANTUS Database Administrator</a>
+                        Contributor: <a href=""></a>
```

Comment:
I couldn't find the `contributor` information in old Cantus json-export, put in a static string as a place holder, and forgot about it. Now I saw it again, I removed the static string so that we will know the ` contributor` field is still unknown.

Reply:
hm... so this remains here effectively as a `TODO` for us? Do we actually want it to be visible to website users?

Reply:
It is visible on old Cantus, but I can't find where that information is stored.